src/platforms/common/usage/index.mdx

```diff
@@ -12,7 +12,7 @@ notSupported:
   - unity
 ---
 
-Sentry's SDK hooks into your runtime environment and automatically reports errors, exceptions, and rejections.
+Sentry's SDK hooks into your runtime environment and automatically reports errors, uncaught exceptions, and unhandled rejections.
```

Comment:
This is a good clarification, thank you. Since it's common content (shared across all platforms), can we update like this:
```suggestion
Sentry's SDK hooks into your runtime environment and automatically reports errors, uncaught exceptions, and unhandled rejections as well as other types of errors depending on the platform.
```